src/secp256k1.js

```diff
@@ -44,10 +44,7 @@ function derivePrivateKey(seed, opts = {}) {
     return privateGen;
   }
   const publicGen = secp256k1.g.mul(privateGen);
-  // A seed can generate many keypairs as a function of the seed and a uint32.
```

Comment:
Maybe a comment elsewhere where we had also used that magic `0`? I think I forgot to make a comment in what's now the deriveNodeAddress function.